components/Chat.js

```diff
@@ -165,19 +156,29 @@ class Chat extends React.Component {
     }
```

Comment:
I am not sure how you are appropriately limiting access to only authenticated users for our services. @inealey @OwenZhang001122